src/include/network/postgres/postgres_command_factory.h

```diff
@@ -1,24 +1,26 @@
 #pragma once
 #include <memory>
-#include "network/abstract_command_factory.h"
 #include "network/postgres/postgres_network_commands.h"
 
+#define MAKE_POSTGRES_COMMAND(type) \
+  std::static_pointer_cast<PostgresNetworkCommand, type>(std::make_shared<type>(packet))
+
 namespace terrier::network {
 
 /**
  * PostgresCommandFactory constructs PostgresNetworkCommands that parses input packets to API calls
  * into traffic cop
  */
-class PostgresCommandFactory : public AbstractCommandFactory {
+class PostgresCommandFactory {
  public:
   /**
    * Convert a Postgres packet to command.
    * @param packet the Postgres input packet
    * @return a shared_ptr to the converted command
    */
-  std::shared_ptr<AbstractNetworkCommand> PacketToCommand(InputPacket *packet) override;
+  virtual std::shared_ptr<PostgresNetworkCommand> PacketToCommand(InputPacket *packet);
```

Comment:
nit: Why did you make these virtual? `PostgresCommandFactory` should probably be a final class

Reply:
I can probably remove the `virtual` keyword. This was just how the old code looks before we added `AbstractCommandFactory`.

Reply:
It might have been virtual for test stub purposes. Weichen would probably know more.

Reply:
@apavlo @GustavoAngulo Now that Tianyu has mentioned about the tests, I notice `network_test.cpp` has a `FakeCommandFactory' that inherits `PostgresCommandFactory` that overrides this function. The `virtual` keyword is therefore needed for the current tests.

Reply:
Ok that makes sense.